homeassistant/components/ihc/binary_sensor.py

```diff
@@ -67,12 +68,16 @@ def __init__(
         self.inverting = inverting
 
     @property
-    def device_class(self):
+    def device_class(self) -> BinarySensorDeviceClass | None:
```

Comment:
This is type hint improvement - unrelated to config-flow. Please move to preliminary/follow-up PR.